templates/frontend/components/header.tpl

```diff
@@ -50,16 +50,12 @@
 					{capture assign="homeUrl"}
 						{url page="index" router=$smarty.const.ROUTE_PAGE}
 					{/capture}
-					{if $displayPageHeaderLogo && is_array($displayPageHeaderLogo)}
+					{if $displayPageHeaderLogo}
 						<a href="{$homeUrl}" class="is_img">
 							<img src="{$publicFilesDir}/{$displayPageHeaderLogo.uploadName|escape:"url"}" width="{$displayPageHeaderLogo.width|escape}" height="{$displayPageHeaderLogo.height|escape}" {if $displayPageHeaderLogo.altText != ''}alt="{$displayPageHeaderLogo.altText|escape}"{/if} />
 						</a>
-					{elseif $displayPageHeaderTitle && !$displayPageHeaderLogo && is_string($displayPageHeaderTitle)}
+					{elseif $displayPageHeaderTitle}
 						<a href="{$homeUrl}" class="is_text">{$displayPageHeaderTitle|escape}</a>
-					{elseif $displayPageHeaderTitle && !$displayPageHeaderLogo && is_array($displayPageHeaderTitle)}
-						<a href="{$homeUrl}" class="is_img">
-							<img src="{$publicFilesDir}/{$displayPageHeaderTitle.uploadName|escape:"url"}" alt="{$displayPageHeaderTitle.altText|escape}" width="{$displayPageHeaderTitle.width|escape}" height="{$displayPageHeaderTitle.height|escape}" />
-						</a>
 					{else}
```

Comment:
I think this last `{else}` statement is not needed, right? There will always be a `$displayPageHeaderTitle`. If so, you can get rid of this last black and make the previous `{elseif}` an `{else}`.

Reply:
I didn't change the logic here. If the logo and title are absent, the OJS logo is shown instead. There may be a situation when the site name is empty.

Reply:
Ah ok, I didn't realize the site name could be empty. :+1:

Reply:
@NateWr, just checked and this applies to the current master branch too: when the second journal is created, a user isn't prompted to a site-level settings page but the page becomes visible on the front-end. Maybe it's a place for improvement?

Reply:
Yeah, my sense is that a site title should be entered on installation. But not a priority for now.